lib/db/lowlevel.go

```diff
@@ -27,7 +27,8 @@ const (
 	// than 100k. For fewer than 100k items we will just get better false
 	// positive rate instead.
 	indirectGCBloomCapacity          = 100000
-	indirectGCBloomFalsePositiveRate = 0.01 // 1%
+	indirectGCBloomFalsePositiveRate = 0.01     // 1%
+	indirectGCBloomMaxMem            = 32 << 20 // Use at most 32MiB memory, which covers our desired FP rate at 27 M items
```

Comment:
I like if mem related variables mention their unit (e.g. here `...MaxMemBytes` or just `...MaxBytes`). Too much potential for confusion, e.g. further down where it needs to be converted to bits (nothing against the usage here, it seems obvious and clear here and now, but in general it isn't necessarily).

Reply:
Done.